src/renderer/styles/about-app.less

```diff
@@ -121,7 +121,30 @@ body {
     font-size: 0.8rem;
     text-align: center;
     padding: 10px 32px;
-    margin: 4px 20px;
+    margin: 4px 8px;
+    display: inline-block;
+    text-decoration: none;
+    line-height: 12px;
+    background-color: rgba(61, 162, 253, 1);
+    color: rgba(255, 255, 255, 1);
```

Comment:
```suggestion
    color: @white;
```

Reply:
this has been removed since it belongs to old design I believe